packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -140,50 +148,63 @@ To remove an item from the cart use `removeItem` method, and similarly to update
 
 To check if a specific product configuration is already in the cart, pass it to `isInCart` method:
 
-```js
-import { computed } from '@vue/composition-api';
-import { useCart } from '{INTEGRATION}';
+```vue
+<template>
+  <!-- ... -->
+    <ul>
+      <li
+         v-for="product in products" :key="product.id"
+      >       
+          <span v-if="isInCart({product})"> added to cart </span>
+      </li>
+    </ul>
+  <!-- ... -->
+</template>    
+<script>
```

Comment:
This is not how you would do this in a real app, I think the UI part can be removed (like it was)

Reply:
removed